qe_metrics/utils/issue_utils.py

```diff
@@ -71,63 +76,73 @@ def update_existing_issue(existing_issue: JiraIssuesEntity, new_issue_data: Issu
                 f'Updating issue "{new_issue_data.key}" in database: "{field}" changed from "{current_value}" to "{new_value}"'
             )
             setattr(existing_issue, field, new_value)
-    orm.commit()
+    db_session.commit()
 
 
 def create_update_issues(
-    issues: List[Issue],
-    product: "ProductsEntity",
-    severity: str,
-    jira_server: str,
+    issues: List[Issue], product: "ProductsEntity", severity: str, jira_server: str, db_session: Session
 ) -> None:
     """
-    Create or update JiraIssuesEntity items in the database from a list of Jira issues. Sets status of obselete issues as "obselete".
+    Create or update JiraIssuesEntity items in the database from a list of Jira issues. Sets status of obsolete issues as "obsolete".
 
     Args:
         issues (List[Issue]): A list of Jira issues
         product (ProductsEntity): A product object
         severity (str): Severity of the issues
         jira_server (str): Jira server URL
+        db_session (Session): SQLAlchemy Session instance.
 
     Returns:
         List["JiraIssuesEntity"]: A list of JiraIssuesEntity objects
     """
     for issue in issues:
-        if existing_issue := JiraIssuesEntity.get(issue_key=issue.key, product=product):
-            update_existing_issue(existing_issue=existing_issue, new_issue_data=issue, severity=severity)
+        if existing_issue := db_session.query(JiraIssuesEntity).filter_by(issue_key=issue.key, product=product).first():
```

Comment:
`db_session.query` returns `Query`; please fix typing

please see the ide warnings about typing (Inspection Lens is a good plugin to use)

Reply:
@rnetser I'm a bit confused about what needs to be fixed in the typing? `db_session.query` does return `Query`, but `db_session.query.first()` returns `(JiraIssuesEntity | None)`.

Thanks for the plugin suggestion.

Reply:
`first() ` return `Any`
`_T = TypeVar("_T", bound=Any)`

`    def first(self) -> Optional[_T]:`

Reply:
resolving for now, if needed we will change it later.